server/models/User.js

```diff
@@ -1,31 +1,40 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 
 const { Schema } = mongoose;
-const bcrypt = require('bcrypt');
-const Event = require('./Event');
+const bcrypt = require("bcrypt");
+const Event = require("./Event");
+const ticketSchema = require("./Ticket");
 
-const userSchema = new Schema({
-  userName : {
-    type: String,
-    required: true,
-    trim: true
+const userSchema = new Schema(
+  {
+    userName: {
+      type: String,
+      required: true,
+      trim: true,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    password: {
+      type: String,
+      required: true,
+      minlength: 5,
+    },
+    events: [Event.schema],
+    tickets: [ticketSchema],
```

Comment:
tickets would not be necessary here, because event schema already contain tickets. We can pull from event directly if we want that piece of information